src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3321,10 +3321,7 @@ Give the compiler some way to differentiate the methods. For example, you can gi
     <value>'id#' syntax is no longer supported. Use '$id' instead.</value>
   </data>
   <data name="WRN_DefineIdentifierRequired" xml:space="preserve">
-    <value>Invalid value for '/define'; '{0}' is not a valid identifier</value>
```

Comment:
><value>Invalid value for a preprocessing symbol; '{0}' is not a valid identifier</value> [](start = 4, length = 88)

I don't think we there is a need to modify this warning. Its wording is fine for command line parser, we simply shouldn't be reusing it for parse options diagnostics because they should be errors instead.